functions/test/generateThumbnail.js

```diff
@@ -0,0 +1,26 @@
+const data  = {
+  bucket:'igv-andamiaje-co.appspot.com',
+  contentDisposition:'inline; filename*=utf-8\'\'tumblr_mmqwshEaAG1qg1dqyo1_1280.jpg',
+  contentType:'image/jpeg',
+  crc32c:'fN3VwQ==',
+  etag:'CI6y37bJhdkCEAE=',
+  generation:'1517517555620110',
+  id:'igv-andamiaje-co.appspot.com/uploads/tumblr_mmqwshEaAG1qg1dqyo1_1280.jpg/1517517555620110',
+  kind:'storage#object',
+  md5Hash:'MTY3ZjAwMWQ3NmYwY2E5NzY4N2EyNTBlYTgzZDRhNzU=',
+  mediaLink:'https://www.googleapis.com/download/storage/v1/b/igv-andamiaje-co.appspot.com/o/uploads%2Ftumblr_mmqwshEaAG1qg1dqyo1_1280.jpg?generation=1517517555620110&alt=media',
+  metadata:{
+    firebaseStorageDownloadTokens:'d4487819-2015-447e-a664-7484af270c0a',
+  },
+  metageneration:'1',
+  name:'uploads/tumblr_mmqwshEaAG1qg1dqyo1_1280.jpg',
+  resourceState:'exists',
+  selfLink:'https://www.googleapis.com/storage/v1/b/igv-andamiaje-co.appspot.com/o/uploads%2Ftumblr_mmqwshEaAG1qg1dqyo1_1280.jpg',
+  size:'252393',
+  storageClass:'STANDARD',
+  timeCreated:'2018-02-01T20:39:15.477Z',
+  timeStorageClassUpdated:'2018-02-01T20:39:15.477Z',
+  updated:'2018-02-01T20:39:15.477Z'
+};
+
+generateThumbnail(data);
```

Comment:
should this test run some kinda of assertation on the return value?

Reply:
Idk what it returns. Would be good to check